ci/firebase_testlab.sh

```diff
@@ -18,22 +18,7 @@ if [[ ! -f "$APP" ]]; then
 fi
 
 GIT_REVISION="${2:-$(git rev-parse HEAD)}"
-BUILD_ID="${3:-$CIRRUS_BUILD_ID}"
```

Comment:
do we know why the logic to check the APK file above failed?  Based on https://flutter.googlesource.com/recipes/+/refs/heads/master/recipes/engine.py#636, the test should have failed, no?

Reply:
It's because `ok_ret=any` a few lines below that. The step was failing, but LUCI was told to accept any exit code.